Dockerfile

```diff
@@ -74,7 +74,11 @@ RUN cget -p $PREFIX install facebook/zstd@v1.4.5 -X subdir -DCMAKE_DIR=build/cma
 RUN cget -p $PREFIX install ccache@v4.1 -DENABLE_TESTING=OFF
 
 # Install newer cmake for onnx runtime
-RUN cget -p /opt/cmake install kitware/cmake@v3.13.4
```

Comment:
Earlier it was keeping binaries in `/opt/rocm`.  Not sure  why.  Does it make a difference for OnnxRuntime if it is now under `/usr/local/` ?

Reply:
I believe was for a system wide install.